### ISSUE TITLE
Add filter error

### ISSUE THREAD

**Issue by hoang-rio** [AdguardTeam/AdguardDNS#369]
### Steps to reproduce

1. Open filter setting
2. Add filter
3. Enter https://abpvn.com/android/abpvn_adguard_dns.txt
4. Click Add filter

### Expected behavior
Filter can add normaly

### Actual behavior
Got server error with log
```
http: panic serving 127.0.0.1:48828: runtime error: index out of range
goroutine 13971 [running]:
net/http.(*conn).serve.func1(0xc0004babe0)
        /usr/local/go/src/net/http/server.go:1746 +0xd0
panic(0x7eb660, 0xcc8c20)
        /usr/local/go/src/runtime/panic.go:513 +0x1b9
main.updateFilter(0xc00044bda0, 0xbee935ceb69adff5, 0xbb5dda05df6, 0xcd6440, 0xc00073da00, 0x0, 0x0)
        /Users/hmage/work/AdguardDNS/control.go:888 +0x9aa
main.refreshFiltersIfNeccessary(0x0)
        /Users/hmage/work/AdguardDNS/control.go:827 +0x2a9
main.handleFilteringAddURL(0xa262a0, 0xc0004a47e0, 0xc0000ebf00)
        /Users/hmage/work/AdguardDNS/control.go:585 +0x423
main.ensure.func1(0xa262a0, 0xc0004a47e0, 0xc0000ebf00)
        /Users/hmage/work/AdguardDNS/helpers.go:31 +0x10b
net/http.HandlerFunc.ServeHTTP(0xc00000bfe0, 0xa262a0, 0xc0004a47e0, 0xc0000ebf00)
        /usr/local/go/src/net/http/server.go:1964 +0x44
net/http.(*ServeMux).ServeHTTP(0xcd6080, 0xa262a0, 0xc0004a47e0, 0xc0000ebf00)
        /usr/local/go/src/net/http/server.go:2361 +0x127
net/http.serverHandler.ServeHTTP(0xc0000696c0, 0xa262a0, 0xc0004a47e0, 0xc0000ebf00)
        /usr/local/go/src/net/http/server.go:2741 +0xab
net/http.(*conn).serve(0xc0004babe0, 0xa26a60, 0xc0002a1900)
        /usr/local/go/src/net/http/server.go:1847 +0x646
created by net/http.(*Server).Serve
        /usr/local/go/src/net/http/server.go:2851 +0x2f5
```

<details><summary>Screenshot:</summary>

</details>

### Your environment

|   Description  |     Value    |
| -------------- | ------------ |
| Version of AdGuard DNS server:| v.v0.1
| How did you setup DNS configuration:| System
| If it's a router or IoT, please write device model:| (e.g. Raspberry Pi 3 Model B)
| Operating system and version:| Centos 7 (Server), Windows 10 (Client)

**Comment by ameshkov** [comment]
Works okay with the version which is currently in the master branch

**Comment by ameshkov** [comment]
Most likely a duplicate of a resolved issue. Plz wait a bit, we're almost ready to publish the stable build of AG.

**Comment by ameshkov** [comment]
@hoang-rio there's a strange issue with this hosts file list: https://abpvn.com/android/abpvn_adguard_dns.txt

It contains lines like `adtimaserver.vn 127.0.0.1` while the proper hosts file must be `127.0.0.1 adtimaserver.vn`.

Another option that works in AdGuard products only would be to omit the IP address and make records look like `||adtimaserver.vn^`. Such a rule will cover subdomains as well.

**Comment by ameshkov** [comment]
The point is that you need to use `https://abpvn.com/android/abpvn.txt` to have these domains blocked

**Comment by dfmcvn** [comment]
This issue has been fixed on v0.1-243-ge356.
Try build latest version with golang and nodejs. :)
